htdocs/core/tpl/card_presend.tpl.php

```diff
@@ -119,6 +119,7 @@
 	// Create form for email
 	include_once DOL_DOCUMENT_ROOT.'/core/class/html.formmail.class.php';
 	$formmail = new FormMail($db);
+	$formmail->setSubstitFromObject($object, $langs);
```

Comment:
The substitutions are managed from lin 212. 
Can you try to move this new line around line 212 so all substitutions management is done at same code emplacement.